docs/Users_Guide/glossary.rst

```diff
@@ -4362,12 +4362,58 @@ METplus Configuration Glossary
      | *Used by:* GenVxMask
 
    GEN_VX_MASK_FILE_WINDOW_BEGIN
-     Used to control the lower bound of the window around the valid time to determine if a GenVxMask input file should be used for processing. Overrides :term:`FILE_WINDOW_BEGIN`. See 'Use Windows to Find Valid Files' section for more information.
+     Used to control the lower bound of the window around the valid time to determine if a GenVxMask input file should
+     be used for processing. Applies to both the input file and the mask file(s).
+     Set :term:`GEN_VX_MASK_OBS_FILE_WINDOW_BEGIN` or
```

Comment:
I see reference to `GEN_VX_MASK_OBS_FILE_WINDOW_BEGIN` here when I was expecting it to be named `GEN_VX_MASK_INPUT_FILE_WINDOW_BEGIN` since I would expect config vars related to the `-input` file to be named `INPUT_FILE` instead of `OBS_FILE`.

Did you use `OBS_FILE` on purpose or did you intend to use `INPUT_FILE` instead?